src/fileActions.jsx

```diff
@@ -202,14 +202,23 @@ const RenameItemModal = ({ path, selected }) => {
         title = _("Rename $0", selected.name);
     }
 
-    const renameItem = () => {
+    const renameItem = async () => {
         const newPath = path.join("/") + "/" + name;
 
-        cockpit.spawn(["mv", "--no-target-directory", path.join("/") + "/" + selected.name, newPath],
-                      { superuser: "try", err: "message" })
-                .then(() => {
-                    Dialogs.close();
-                }, err => setErrorMessage(err.message));
+        try {
+            await cockpit.spawn([
+                "mv", "--interactive", "--no-target-directory",
+                path.join("/") + "/" + selected.name, newPath
+            ],
+                                { superuser: "try", err: "message", environ: ["LC_ALL=C"] }).input();
+            Dialogs.close();
+        } catch (err) {
+            if (err.message.includes("overwrite")) {
```

Comment:
I don't like this.  I was going to propose checking if the message ends with `"?"` but you might not like that either, so how about this:

Let's do `.input('n\n');` and then:
  - success case: clean stderr, exit code 0
  - file exists case: stderr message, exit code 0
  - other error: non-0 exit

Reply:
This does not hold up:
```
[admin@fedora-40-127-0-0-2-2201 ~]$ mv -i foo bar
mv: overwrite 'bar'? n
[admin@fedora-40-127-0-0-2-2201 ~]$ echo $?
1
```

Reply:
Ugh :(

Reply:
I think the solution in the copy PR is acceptable as compromise:

https://github.com/cockpit-project/cockpit-files/pull/457/files#diff-1f8c6aa1fae7f48ccf3ee10b560d6111e12cc582bc158ee3c0367871f6387582R484